src/sonic-host-services/scripts/hostcfgd

```diff
@@ -9,7 +9,7 @@ import syslog
 
 import jinja2
 from sonic_py_common import device_info
-from swsssdk import ConfigDBConnector
+from swsscommon import swsscommon
```

Comment:
Can we just import only ConfigDBConnector if we don't need whole swsscommon module then we don't need any other changes in this file?
from swsscommon.swsscommon import ConfigDBConnector

Reply:
Resolved.